### PR TITLE
AC-679: Create phpcs static check for ObsoleteSystemConfigurationTest

### PR DESCRIPTION
**Pull request by eliseacornejo**

### CODE REVIEW

File: Magento2/Sniffs/Legacy/ObsoleteSystemConfigurationSniff.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        }
+    }
+
+    /**
+     * Adds an invalid XML error
+     *
+     * @param File $phpcsFile
+     * @param int $stackPtr
+     */
+    protected function invalidXML(File $phpcsFile, int $stackPtr): void
```
Comment (jcuerdo) [inline Magento2/Sniffs/Legacy/ObsoleteSystemConfigurationSniff.php:66]: Here private is enough.

**Review by jcuerdo — DISMISSED** [review]

File: Magento2/Sniffs/Legacy/ObsoleteSystemConfigurationSniff.php [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+class ObsoleteSystemConfigurationSniff implements Sniff
+{
+    private const ERROR_CODE_XML = 'WrongXML';
+    private const WARNING_CODE_OBSOLETE = 'FoundObsoleteSystemConfiguration';
+
+    /**
+     * @inheritdoc
+     */
+    public function register()
```
Comment (loginesta) [inline Magento2/Sniffs/Legacy/ObsoleteSystemConfigurationSniff.php:22]:
```suggestion
     public function register(): array
```

**Review by loginesta — APPROVED** [review]

**Review by loginesta — APPROVED** [review]

**Comment by eliseacornejo** [comment]
@magento import pr to magento-commerce/magento-coding-standard

**Comment by m2-github-services** [comment]
@eliseacornejo the Pull Request is successfully imported.

**Pull request merged**